File: src/main/java/net/guneyilmaz0/permissionss/event/PlayerRankExpiredEvent.java
package net.guneyilmaz0.permissionss.event;

import cn.nukkit.Player;
import cn.nukkit.event.plugin.PluginEvent;
import lombok.Getter;
import net.guneyilmaz0.permissionss.Main;

@Getter
public class PlayerRankExpiredEvent extends PluginEvent {
    private final Player player;

    public PlayerRankExpiredEvent(Player player) {
        super(Main.getInstance());
        this.player = player;
    }
}

File: src/main/kotlin/events/PlayerRankExpiredEvent.kt
package net.guneyilmaz0.permissions.events

import cn.nukkit.Player
import cn.nukkit.event.plugin.PluginEvent
import net.guneyilmaz0.permissions.PermissionsS

class PlayerRankExpiredEvent(val player: Player) : PluginEvent(PermissionsS.instance)
